src/porepy/geometry/map_geometry.py

```diff
@@ -37,9 +36,9 @@ def force_point_collinearity(
 
     return pts[:, 0, np.newaxis] * (1 - dist) + pts[:, end, np.newaxis] * dist
 
-
 def map_grid(
-    sd: pp.Grid, tol: float = 1e-5, R=None
+    sd: pp.Grid, tol: Optional[float] = 1e-5, 
```

Comment:
```suggestion
    sd: pp.Grid, tol: float = 1e-5, 
```
Optional should only be used if the default is None.